docs/governance/politeia/politeia-censorship.md

```diff
@@ -0,0 +1,28 @@
+# <img class="dcr-icon" src="/img/dcr-icons/Politeia.svg" /> Politeia Censorship
+
+---
+
+This page details the Politeia censorship model, its motivations, and provides instructions for proving censorship of submissions to Politeia (proposals, proposal comments, comment upvotes/downvotes).
+
+## Why censorship is required
+
+Censorship is a necessary tool for administrators for several reasons. For one, Politeia cannot host illegal content. For another, because the fee to submit a proposal is relatively low (0.1 DCR), there is a risk the system could be flooded with spam proposals that waste the time and attention of stakeholders. 
```

Comment:
I would word this differently.  Admins are not suppose to be passing judgement on the quality of a proposal.  The only time admins should be censoring proposals is if the proposal does not meet the proposal guidelines.  Things like containing illegal content, not being formatted correctly, or not containing the information that all proposals are required to have.

I'm not sure if the exact proposal guidelines are written out anywhere.  If they're not, that's something that should be added to the docs.

Reply:
Have we reached consensus on this? You would know more than me, but following some of the conversations in #proposals, I got the sense that low quality proposals (i.e. spam) were being allowed through for now to test the system, but that we might censor spam in the future. This idea has also been expressed, for instance, in the Medium piece @RichardRed0x wrote on [Politeia's launch](https://medium.com/decred/on-the-launch-of-decreds-politeia-b305d06d8bda) .

> Anyone who pays a small fee ( 0.1 DCR) can register an account and submit proposals. Proposals are screened by admins to remove inappropriate content (illegal or spam).). 

Would we, for instance, allow a proposal suggesting Decred invest in a multi-level marketing scheme selling Cialis? 

I know some time has passed though since launch, and I've missed some convos. If our rules have firmed up on this, I think your suggested rewording is good. 

> I'm not sure if the exact proposal guidelines are written out anywhere. If they're not, that's something that should be added to the docs.

We do have a [proposal guidelines page](https://docs.decred.org/governance/politeia/proposal-guidelines/). We created this page a while ago though, and purposely didn't get too specific on the required structure of a proposal. The rationale being that we couldn't know what would be the best stucture/requirement set until we put a few proposals through the system. Some proposals have followed our (very high level) template (What? Why? When? etc.). Others (including some put forward by Decred contributors) follow a more traditional proposal structure (Abstract, Motivation, etc.), but do answer the basic questions. Has there been a firming up of proposal requirements?

Reply:
I think a lot of this stuff is still being figured out.  I'm not sure of the answers myself.  I would say to word it however you think is best and we can always update in the future once things get a bit more hashed out.